auction9-frontend/src/app/services/auction.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuctionService {
+
+  constructor(private http: HttpClient) { }
+
+  // base endpoint
+  private readonly baseEndpoint = window.location.protocol + '//' + window.location.hostname + ':3000';
```

Comment:
Ok for now, but this should be part of frontend configuration, so we can have multiple environments